builtin/mainmenu/tab_about.lua

```diff
@@ -71,33 +66,51 @@ local previous_core_developers = {
 	"Ryan Kwolek (kwolekr) <kwolekr@minetest.net>",
 	"sapier",
 	"Zeno",
-	"ShadowNinja <shadowninja@minetest.net>",
 	"Auke Kok (sofar) <sofar@foo-projects.org>",
 	"Aaron Suen <warr1024@gmail.com>",
+	"paramat",
+	"Pierre-Yves Rollo <dev@pyrollo.com>",
 }
 
 local previous_contributors = {
-	"Nils Dagsson Moskopp (erlehmann) <nils@dieweltistgarnichtso.net> [Minetest Logo]",
+	"Nils Dagsson Moskopp (erlehmann) <nils@dieweltistgarnichtso.net> [Minetest logo]",
 	"red-001 <red-001@outlook.ie>",
 	"Giuseppe Bilotta",
+	"ClobberXD",
 	"Dániel Juhász (juhdanad) <juhdanad@gmail.com>",
 	"MirceaKitsune <mirceakitsune@gmail.com>",
 	"Constantin Wenger (SpeedProg)",
 	"Ciaran Gultnieks (CiaranG)",
 	"Paul Ouellette (pauloue)",
 	"stujones11",
+	"srfqi",
 	"Rogier <rogier777@gmail.com>",
 	"Gregory Currie (gregorycu)",
 	"JacobF",
-	"Jeija <jeija@mesecons.net> [HTTP, particles]",
+	"Jeija <jeija@mesecons.net>",
 }
 
-local function buildCreditList(source)
+local function prepare_credits(dest, source)
+	for _, s in ipairs(source) do
+		-- if there's text inside brackets make it gray-ish
+		s = s:gsub("(%[.+%])", function(s2)
+			return core.colorize("#aaa", s2)
+		end)
+		dest[#dest+1] = s
+	end
+end
+
+local function build_hacky_list(items, spacing)
+	spacing = spacing or 0.5
+	local y = spacing / 2
 	local ret = {}
-	for i = 1, #source do
-		ret[i] = core.formspec_escape(source[i])
+	for _, item in ipairs(items) do
+		if item ~= "" then
+			ret[#ret+1] = "label[0," .. y .. ";" .. core.formspec_escape(item) .. "]"
```

Comment:
Concatenating `y` when MT expects a number in `%f` format might be error-prone - I suggest explicitly using `%f` here since (1) number tostringing has no format guarantees in Lua (2) what might theoretically happen (and sometimes happens, but usually only for very small / large numbers where exp notation is more compact) is that Lua uses scientific notation (equivalent to `%g`) which I don't think MT supports.

In practice this usually works though.